chains/evm/executor/executor.go

```diff
@@ -82,11 +74,11 @@ func NewExecutor(
 }
 
 // Execute starts a signing process and executes proposals when signature is generated
-func (e *Executor) Execute(msgs []*message.Message) error {
+func (e *Executor) Execute(proposals []*proposal.Proposal) error {
+
```

Comment:
Can you please look at the executor.go diff and remove all the extra whitespace changes that were added.